consensus-types/blocks/factory.go

```diff
@@ -56,6 +56,12 @@ func NewSignedBeaconBlock(i interface{}) (interfaces.SignedBeaconBlock, error) {
 		return initBlindedSignedBlockFromProtoCapella(b.BlindedCapella)
 	case *eth.SignedBlindedBeaconBlockCapella:
 		return initBlindedSignedBlockFromProtoCapella(b)
+	case *eth.GenericSignedBeaconBlock_Deneb:
+		return initSignedBlockFromProtoDeneb(b.Deneb.Block)
+	case *eth.SignedBeaconBlockDeneb:
+		return initSignedBlockFromProtoDeneb(b)
+	case *eth.SignedBlindedBeaconBlockDeneb:
+		return initBlindedSignedBlockFromProtoDeneb(b)
```

Comment:
Missing `GenericSignedBeaconBlock_BlindedDeneb`

If everything works correctly without this, it makes me wonder if we even need `GenericXXX` cases here. But that it a question for another time, here I would blindly add it.